app/src/main/java/org/mozilla/tv/firefox/pinnedtile/PinnedTileRepo.kt

```diff
@@ -42,6 +46,11 @@ class PinnedTileRepo(private val applicationContext: Application) {
         return _pinnedTiles
     }
 
+    @VisibleForTesting(otherwise = VisibleForTesting.PROTECTED)
+    fun getPinnedTiles(): MutableLiveData<LinkedHashMap<String, PinnedTile>> {
+        return _pinnedTiles
```

Comment:
Nit: I'd prefer adding @VisibleForTesting to `val _pinnedTiles` and removing this function entirely

Reply:
Agreed - though it looks like you're using pinnedTiles in the model. In that case, can you make this return type `LiveData` rather than `MutableLiveData` to prevent any future callers from modifying the list from this reference?